File: tests/py-files/test-struct2.py
import sys

class Cars:
    def __init__(self, topspeed = 0, acceleration = 0, Tier = 0):
        self.topspeed = topspeed
        self.acceleration = acceleration
        self.Tier = Tier
class Result:
    def __init__(self, Maths = 0, Overall_Grade = 0):
        self.Maths = Maths
        self.Overall_Grade = Overall_Grade
def main():

    benz = Cars()
    benz.topspeed = 200
    benz.acceleration = 4
    benz.Tier = 'A'
    print("Benz is {} grade and top speed is {}, will reach in {} second".format( benz.Tier,benz.topspeed,benz.acceleration))
    Toby = Result()
    Toby.Maths = 50
    print("Toby got {}, in Math".format(Toby.Maths))
    return 0

if __name__ == '__main__':
    main()

File: tests/c-files/test-struct2.c
#include <stdio.h>

struct Cars
{
    int topspeed;
    int acceleration;
    char Tier;
};

union Result
{
    int Maths;
    char Overall_Grade;
};

int main()
{
    struct Cars benz;
    benz.topspeed = 200;
    benz.acceleration = 4;
    benz.Tier = 'A';
    printf("Benz is %c grade and top speed is %d, will reach in %d second\n", benz.Tier,benz.topspeed,benz.acceleration);



    union Result Toby;
    Toby.Maths = 50;


    printf("Toby got %d, in Math\n",Toby.Maths);
    return 0;
}
